frontend/http/writer_test.go

```diff
@@ -1,15 +1,12 @@
-// Copyright 2016 The Chihaya Authors. All rights reserved.
-// Use of this source code is governed by the BSD 2-Clause license,
-// which can be found in the LICENSE file.
-
 package http
 
 import (
 	"net/http/httptest"
 	"testing"
 
-	"github.com/chihaya/chihaya/tracker"
 	"github.com/stretchr/testify/assert"
```

Comment:
Alright, let's not make changes now and later upgrade them to `require` all at once.